core/scripts/cli/sponsor/redeem.js

```diff
@@ -1,15 +1,21 @@
 const inquirer = require("inquirer");
 const getDefaultAccount = require("../wallet/getDefaultAccount");
-const { unwrapToEth } = require("./currencyUtils.js");
+const { unwrapToEth, getIsWeth, getCurrencySymbol } = require("./currencyUtils.js");
 const { submitTransaction } = require("./transactionUtils");
 
 const redeem = async (web3, artifacts, emp) => {
+  const ExpandedERC20 = artifacts.require("ExpandedERC20");
   const { fromWei, toWei, toBN } = web3.utils;
   const SyntheticToken = artifacts.require("SyntheticToken");
   const sponsorAddress = await getDefaultAccount(web3);
   const collateral = (await emp.getCollateral(sponsorAddress)).toString();
   const position = await emp.positions(sponsorAddress);
 
+  const collateralCurrency = await ExpandedERC20.at(await emp.collateralCurrency());
+  const isWeth = await getIsWeth(web3, artifacts, collateralCurrency);
+  const collateralSymbol = await getCurrencySymbol(web3, artifacts, collateralCurrency);
+  const requiredCollateralSymbol = isWeth ? "ETH" : collateralSymbol;
```

Comment:
How do you choose when to use "WETH" vs "ETH" in printouts?

Reply:
The original aim was to quote collateral values to the user in ETH always. And then (un)wrap for them, so only use WETH when describing the transactions themselves.

Though as Nick pointed out above, maybe this isn't worth it at all.

Reply:
I think ^ makes sense. That was roughly my intuition after reading, just wanted to make sure I was thinking about it the right way.